InaccessibleObjectException when running a Flow test project using hotswap agent

```
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make public void jdk.internal.loader.AbstractClassLoaderValue.removeAll(java.lang.ClassLoader) accessible: module java.base does not "exports jdk.internal.loader" to unnamed module @18df8434
```

Probably needs yet another `--add-opens`

--- a/src/main/kotlin/com/vaadin/plugin/hotswapagent/HotswapAgentProgramPatcher.kt
+++ b/src/main/kotlin/com/vaadin/plugin/hotswapagent/HotswapAgentProgramPatcher.kt
@@ -50,6 +50,8 @@ class HotswapAgentProgramPatcher : JavaProgramPatcher() {
         paramsList.add("java.base/sun.security.action=ALL-UNNAMED")
         paramsList.add(addOpens)
         paramsList.add("java.desktop/java.beans=ALL-UNNAMED")
+        paramsList.add(addOpens)
+        paramsList.add("java.base/jdk.internal.loader=ALL-UNNAMED")
 
         paramsList.add("-XX:+AllowEnhancedClassRedefinition")
         paramsList.add("-XX:+ClassUnloading")

